### ISSUE TITLE
Typescript issue with PlaylistItem

### ISSUE THREAD

**Issue by porridgewithraisins** [talmobi/yt-search#67]
```   
 interface PlaylistItem {
        title: string;
        videoId: string;
        listId: string;
        thumbnail: string;
        author: Author;
    }
```

`Property 'duration' is missing in type 'PlaylistItem'`
But, duration comes up when I log `(await yts(...)).videos`, And I want to use the field

Workaround:
I extended PlaylistItem into my own interface.

Do the types need to be fixed? or am I missing something.

thank you

**Comment by talmobi** [comment]
@porridgewithraisins Hello~

I don't think this library comes with typescript definitions. I think the codebase you are working with has made their own interface already.

**Comment by porridgewithraisins** [comment]
You're right, I was using https://www.npmjs.com/package/@types/yt-search and presumed you were the one that made it. I now see it was made by a different individual. Sorry for the bother

**Comment by talmobi** [comment]
@porridgewithraisins oh that's very cool! I don't know how that is managed, and I won't pursue them to update it, but seems like some kind of big repository of types for many packages. Thanks for looking into it and replying back~
